### ISSUE TITLE
build faild when there is a space in project name

### ISSUE THREAD

**Issue by jovirus** [CocoaPods/CocoaPods#6316]
Hey guys,
I have the latest pod installed, but when I build my project. At last build step it will ended up in an error 
"/project_name../No such file or directory found"

**Comment by joedaniels29** [comment]
I'm experiencing this also. its right here I think the issue is happening 
```bash
local code_sign_cmd="/usr/bin/codesign --force --sign ${EXPANDED_CODE_SIGN_IDENTITY} ${OTHER_CODE_SIGN_FLAGS} --preserve-metadata=identifier,entitlements "$1""
```

**Comment by benasher44** [comment]
Hi @jovirus! Thanks for reporting this! There's actually already a fix for this on master (#6160). An RC version for 1.2.0 should be released soon, which will include the fix. Have a nice day!
